src/Select2/Select2.less

```diff
@@ -94,15 +115,23 @@
   .text--medium();
 }
 
+.Select2--trailingElement {
+  .margin--left--xs();
+  min-height: @size_2xl;
+}
+
+.Select2--errorIcon {
+  color: @alert_red;
+  display: flex;
+  align-items: center;
```

Comment:
I notice that the flex mixins are used in a rule at the top of the file. Is there a particular reason for using native css flex here? I'm not suggesting one over the other, rather just wondering if it was intentional.

Reply:
whoops missed it, yeah. Thanks.

We need the flex mixins for cross-browser support. I think fampo set it up so we could use the native attributes and to autogenerate the platform-specific attributes

(we should use that at some point)

Reply:
Ah that's right, I forgot what the mixins did besides save us from typing a few characters 😄